examples/js/exporters/GLTFExporter.js

```diff
@@ -1165,6 +1165,30 @@ THREE.GLTFExporter.prototype = {
 				var attribute = geometry.attributes[ attributeName ];
 				attributeName = nameConversion[ attributeName ] || attributeName.toUpperCase();
 
+				// Prefix all geometry attributes except the ones specifically
+				// listed in the spec; non-spec attributes are considered custom.
+				var validVertexAttributes = [
+					/^POSITION$/,
+					/^NORMAL$/,
+					/^TANGENT$/,
+					/^TEXCOORD_\d+$/,
+					/^COLOR_\d+$/,
+					/^JOINTS_\d+$/,
+					/^WEIGHTS_\d+$/,
+				];
+
+				var isValidAttribute = false;
+				for (var i = 0; i < validVertexAttributes.length; i++) {
+					if (validVertexAttributes[i].test(attributeName)) {
+						isValidAttribute = true;
+						break;
+					}
+				}
+				if (!isValidAttribute) {
+					console.log(`Prefixing ${attributeName}`)
```

Comment:
Probably no need to log anything here, but if we did it should be more self-explanatory:

```
THREE.GLTFExporter: Renamed attribute ${lowecaseAttributeName} as ${newAttributeName}.
```

Reply:
Excellent point -- thanks. I agree, no need to log (there could be thousands). I'll take care of these issues shortly.